README.md

```diff
@@ -95,13 +94,10 @@ This SDK abides by our Client SDK Deprecation Policy. For more information on th
 | 5.x.x | Inactive | February 2021 | June 2024 | June 2025 |
 | 4.x.x | Unsupported | November 2015 | February 2022 | February 2023 |
 
-Versions 4.9.6 and below use outdated SSL certificates and are unsupported.
```

Comment:
This was for the _old_ cert rotation so removing since v4 has been unsupported for over a year. There is also a section above about the new certs.